frontend/pages/hosts/details/cards/Software/SelfService/components/InstallAllInCategoryButton/InstallAllInCategoryButton.tests.tsx

```
@@ -163,6 +163,101 @@ describe("InstallAllInCategoryButton", () => {
+      await user.click(
+        screen.getByRole("button", { name: /Install all \(3\)/i })
+      );
+      await user.click(
+        await screen.findByRole("button", { name: /^Install all$/i })
+      );
+
+      await waitFor(() => {
+        expect(requestedUrl).not.toContain("query=");
+      });
+      unmount();
+    }
```

ESLint

[error] iterators/generators require regenerator-runtime, which is too heavyweight for this guide to allow them. Separately, loops should be avoided in favor of array iterations.

(no-restricted-syntax)